src/main/java/nl/thehyve/ocdu/factories/PatientDataFactory.java

```diff
@@ -82,4 +81,31 @@ private void setValue(String[] row, Map<String, Integer> columnsIndex, String co
     }
 
 
+    public List<String> generatePatientRegistrationTemplate(MetaData metadata, Map<String, String> subjectMap, boolean registerSite) {
+        List<String> result = new ArrayList<>();
+        String delim = "\t";
+        String header = "Study Subject ID"+delim;
+        if(metadata.isGenderRequired()) header += "gender"+delim;
```

Comment:
here instead of doing string concatenation you can just add string to a list called lets say headers, the fact that there are conditional statements does not go against it. 
Then you just call String.join with delimiter only once, instead of using delim in each line.
Plus: if you se ALT+CMD+L you can avoid styling errors like lack of spaces between operators and variables.